File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main ()
{
    int n;
    do
    {
       // i ask for one number between 1 and 8
        n = get_int("Height: ");
    }
    while (!((n>0) && (n<9)));
    for (int i=0; i<n; i++)
    {
        for (int p=n-1; p>i; p--)
        {
            printf(" ");
        }
        for (int j=-1; j<i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Mario/less/mario.py
from cs50 import get_int


def main():
    # I ask the user for a number
    Heigth = get_int("Give a number between 1 and 8 for the heigth of the pyramid: ")
    
    # I recursivly call the main function to be sure the user promp the needed information
    if (Heigth > 0) and (Heigth < 9):
        PrintPyramid(Heigth)
    else:
        main()
        

def PrintPyramid(Heigth):
    # I declared variables
    Blank = (Heigth - 1)
    
    # While loop
    for L in range(0, Heigth, 1):
        # The blank spaces nedeed in every loop
        for b in range(Blank):
            print(" ", end="")
        # The blocks spaces nedeed in every loop
        for i in range(0, (L+1), 1):
            print("#", end="")
        print("")
        Blank -= 1

        
if __name__ == "__main__":
    main()